taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -139,16 +142,14 @@ const Entry = () => {
                 </Stack>
                 <Stack direction="row" alignItems="center" sx={{m: 2}}>
                     <Typography>Main Language</Typography>
-                    <TextField
-                        onChange={(e) => { 
-                            setNewLanguageCode(e.target.value);
-                            setIsValidLanguageCode(ISO6391.validate(e.target.value));
+                    <Autocomplete
+                        options={languageOptions}

```

Comment:
As I mentioned above, you can directly give this list to options.
```suggestion
                        options={Object.keys(ISO_CODES)}

```